lib/matplotlib/collections.py

```diff
@@ -274,11 +274,11 @@ def get_datalim(self, transData):
                 # can properly have the axes limits set by their shape +
                 # offset.  LineCollections that have no offsets can
                 # also use this algorithm (like streamplot).
-                result = mpath.get_path_collection_extents(
-                    transform.get_affine(), paths, self.get_transforms(),
+                return mpath.get_path_collection_extents(
```

Comment:
again, this could get some more explanation, given that there is a comment above, but this is clearly different.  I'm not even sure I understand why you can change this like this.

Reply:
The first argument is the master transform; instead of doing one transform when finding extents, and a second one after on the result, this does a combined transform from the get-go. This ensures that whatever is calculated for `minpos` is in the final coordinate space.